brion/target.h

```diff
@@ -67,6 +67,7 @@ class Target
      *
      * @param name target name to get the values from
      * @return list of values in the target. Can be targets and/or GIDSet
+     * @throw std::runtime_error if name is an invalid target
```

Comment:
If this throws, Target::parse should also throw. The funny part is that you've implemented the throw in parse but not in get.